File: crates/cccc-cli/tests/fixtures/sample.zig
pub fn sumOfPrimes(max: u32) u32 {
    var total: u32 = 0;
    outer: for (2..max) |i| {
        for (2..i) |j| {
            if (i % j == 0) {
                continue :outer;
            }
        }
        total += i;
    }
    return total;
}

File: crates/cccc-cli/tests/fixtures/sample.dart
int sumOfPrimes(int max) {
  var total = 0;
  outer:
  for (var i = 2; i <= max; ++i) {
    for (var j = 2; j < i; ++j) {
      if (i % j == 0) {
        continue outer;
      }
    }
    total += i;
  }
  return total;
}
